### DISCUSSION TITLE
click and open posts

### DISCUSSION THREAD

**Discussion by AnonymousWebHacker** [hyperspacedev/hyperspace#241] [Ideas]
I think it would be ideal, like mastodon does, twitter ..etc
If you click on a post, it should open the post with its image, and toots related to that post

**Comment by alicerunsonfedora** [comment]
We've had this behavior before in Hyperspace Desktop Classic, where tapping anywhere on a post would take you directly to that post from the timeline. However, the implementation for this was hacked together at best and hard to maintain at worst. For now, you can tap on "Show thread" to imitate this behavior, but I don't believe we have a better means of implementing this behavior.

**Reply by AnonymousWebHacker** [reply]
Thank you for your quick response to my questions.
